float64 x
float64 y
float64 heading
int32 index
float32 target_speed
float32 target_brake
float32 target_gear
float32 speed
float32 steer
int32 brake
int16 gear
int16 auto_manual
string map_folder
string map_file